cmd/fund/cmd.go

```diff
@@ -0,0 +1,87 @@
+package fund
+
+import (
+	"errors"
+
+	_ "embed"
+
+	"github.com/maticnetwork/polygon-cli/util"
+	"github.com/spf13/cobra"
+)
+
+// defaultPrivateKey is the default private key used to fund wallets.
+const defaultPrivateKey = "0x42b6e34dc21598a807dc19d7784c71b2a7a01f6480dc6f58258f78e539f1a1fa"
+
+// cmdParams holds the command-line parameters for the fund command.
+type cmdFundParams struct {
+	RpcUrl     *string
+	PrivateKey *string
+
+	WalletCount            *uint64
+	WalletFundingHexAmount *string
+	OutputFile             *string
+
+	FunderAddress *string
+}
+
+var (
+	//go:embed usage.md
+	usage  string
+	params cmdFundParams
+)
+
+// FundCmd represents the fund command.
+var FundCmd = &cobra.Command{
+	Use:   "fund",
+	Short: "Bulk fund crypto wallets automatically.",
+	Long:  usage,
+	PreRunE: func(cmd *cobra.Command, args []string) error {
+		return checkFlags()
+	},
+	RunE: func(cmd *cobra.Command, args []string) error {
+		return runFunding(cmd.Context())
+	},
+}
+
+func init() {
+	p := new(cmdFundParams)
+	flagSet := FundCmd.Flags()
+
+	p.RpcUrl = flagSet.StringP("rpc-url", "r", "http://localhost:8545", "The RPC endpoint url")
+	p.PrivateKey = flagSet.String("private-key", defaultPrivateKey, "The hex encoded private key that we'll use to send transactions")
+
+	// Wallet parameters.
+	p.WalletCount = flagSet.Uint64P("wallets", "w", 10, "The number of wallets to fund")
+	p.WalletFundingHexAmount = flagSet.StringP("amount", "a", "0xb1a2bc2ec50000", "The amount of wei to send to each wallet") // 0xb1a2bc2ec50000 represensts 0.05 ether.
```

Comment:
nit: just my opinion, but this change feels less intuitive for users. in my mind, user will want to specify amt in eth as human readable format for quicker iteration

Reply:
Yep, I wanted to have the same behaviour on `polycli fund` and `polycli loadtest` but you're right, it's easier to use amount in eth format. Will update this asap.

Reply:
It's better like this :)

```bash
$ go run main.go fund --help | grep eth-amount
  -a, --eth-amount float        The amount of ether to send to each wallet (default 0.05)

$ go run main.go loadtest --help | grep eth-amount
      --eth-amount float                       The amount of ether to send on every transaction (default 0.001)
```